### ISSUE TITLE
password support in $MPD_HOST

### ISSUE THREAD

**Issue by mhuesch** [boysetsfrog/vimpc#101]
hi, thanks for this great project! I have been using it for a while and really liking it :slightly_smiling_face: 

I just added auth to my `mpd` (so I could use a remote control securely) and found what I think is a slight deviation in behavior around env vars between `mpc` and `vimpc`. wanted to post it here in case it's helpful to anyone else debugging, even if implementing a fix is decided against (it's quite to work around using the `--host` flag in a shell alias, e.g. `alias vimpc ="vimpc --host $MPD_HOST"`).

basically, setting `MPD_HOST` to `password@localhost` allows `mpc` to run successfully, whereas `vimpc` complains "You need a password". so `mpc` must do some conditional parsing there and support both `password@host` and `host` forms.

this is very minor, just thought it might be worth bringing up.
